excercises/3-synchronization/handling-errors.spec.ts

```diff
@@ -12,55 +12,70 @@ describe('angularjs homepage', function () {
   })
 
   it('should not fail if element is not present', async function () {
```

Comment:
also I understand that the test should not fail if browser will not find the element. I ran this test and it is failing.